content/docs/user-guide/interactivity/input/using-player-input.md

```diff
@@ -0,0 +1,145 @@
+
+### Using input in Lua 
+
+You can also create a Lua script that connects to input events. For more information about scripting with Lua, refer to [Writing Lua Scripts](/docs/user-guide/scripting/lua).
+
+1. In O3DE Editor, select the entity with the Input component in the viewport or **Entity Outliner**.
+
+1. In Entity Inspector, click **Add Component**, and then add the **Lua Script** component.
+
+1. In the Lua Script component, click {{< icon "open-in-internal-app.svg" >}}.
+
+1. In Lua Editor, create a table of the functions `OnPressed`, `OnHeld`, and `OnReleased` for an input event that you want to connect the script to.
+
+1. Create a bus handler by connecting to the `InputEventNotificationBus` with the first parameter set to the table of functions for the input event. The second parameter is an `InputEventNotificationId` for the **Event Name**.
+
+### Example Lua script
+
+The following Lua script connects `Action` events to the individual functions `OnPressed`, `OnHeld`, and `OnReleased`.
+
+```lua
+local tutorial_input = {
+    Properties = {}
+}
+function tutorial_input:OnActivate()
+    self.ActionInputs = {}
+
+    self.ActionInputs.OnPressed = function(_, value)
+        TransformBus.Event.SetLocalUniformScale(self.entityId, 2.0)
+    end
+
+    self.ActionInputs.OnHeld = function(_, value)
+        TransformBus.Event.RotateAroundLocalZ(self.entityId, 0.01)
+    end
+
+    self.ActionInputs.OnReleased = function(_, value)
+        TransformBus.Event.SetLocalUniformScale(self.entityId, 1.0)
+    end
+
+    self.ActionInputHandler = InputEventNotificationBus.Connect(self.ActionInputs, InputEventNotificationId("Action"))
+end
+
+function tutorial_input:OnDeactivate()
+    self.ActionInputHandler:Disconnect()
+end
+
+return tutorial_input
+```
+
+## Testing player input
+
+After you use either Script Canvas or Lua to connect the input event in your `.inputbindings` file to an input handler, you can test player input in **O3DE Editor**.
+
+1. In O3DE Editor, select the entity with the Input and scripting components in the viewport or **Entity Outliner**.
+
+    {{< note >}}
+In this example, the entity must have a component that enables visibility in the game, such as the [Mesh component](/docs/user-guide/components/reference/atom/mesh), the [White Box component](/docs/user-guide/components/reference/shape/white-box), or one of the [Shape components](/docs/user-guide/components/reference/shape).
+{{< /note >}}
+
+1. In Entity Inspector, click **Add Component**, and then add the **Mesh** component.
+
+1. For **Mesh asset**, specify a mesh asset file. This gives your entity a shape that is visible in game mode.
+
+1. To enter game mode, press **Ctrl+G**.
+
+1. Press **Space bar** so that your entity rotates on the local z-axis. The entity's scale doubles when the event `OnPressed` is received from the input handler that is connected to `Action`. The entity's scale returns to normal when the `OnReleased` event is received.
```

Comment:
Following up from [our previous discussion](https://github.com/o3de/o3de.org/pull/998#discussion_r728519987), try to reword this to avoid passive voice ("is received").

>any thoughts on formatting for SC nodes though?

To refer to a specific SC node, I'd go with "**[NodeName]** node", as you did in line 76 ("the **InputHandler** node").